Bindings/Java/Matlab/Utilities/osimC3D.m

```diff
@@ -81,11 +120,86 @@ function rotateData(obj,axis,value)
             % rotate the tables
             obj.rotateTable(obj.markers, axis, value);
             obj.rotateTable(obj.forces, axis, value);
-            
             disp('Marker and Force tables have been rotated')
         end
+        function writeTRC(obj,varargin)
+            % Write marker data to trc file.
+            % osimC3d.writeTRC()                       Write to current dir using c3d name
+            % osimC3d.writeTRC('Walking.trc')          Write to current dir with input name
+            % osimC3d.writeTRC('C:\data\Walking.trc')  Write to input location from full input path
+
+            % Compute an output path to use for writing to file
+            outputPath = generateOutputPath(obj,varargin,'.trc');
+
+
+            import org.opensim.modeling.*
+            % Write to file
+            TRCFileAdapter().write( obj.getTable_markers, outputPath)
+            disp(['Marker file written to ' outputPath]);
+        end
+        function writeMOT(obj,varargin)
+         % Write forces data to mot file
+         % osimC3d.writeMOT()                       Write to current dir using c3d name
+         % osimC3d.writeMOT('Walking.mot')          Write to current dir with input name
```

Comment:
I'm not sure the invocations above write to the current dir. I think these invocations write the TRC file next to the C3D file (`obj.getPath()`), wherever it is.